packages/augur-ui/src/modules/market/components/core-properties/core-properties.tsx

```diff
@@ -182,6 +161,27 @@ const CoreProperties: React.FC<CorePropertiesProps> = ({
               }
             </>
           )}
+          {loadAffiliateFee && <PropertyLabel
```

Comment:
Won't this insert the falsy `loadAffiliateFee` into the children of the parent dom node? Like, wouldn't that become an errant `0` or `undefined` or something?

Reply:
I'll take any false
undefined | 0 | false

as long as Function returns true.